website/docs/cloud-docs/api-docs/audit-trails.mdx

```diff
@@ -36,7 +36,7 @@ page_title: Audit Trails - API Docs - Terraform Cloud
 
 # Audit Trails API
 
-Terraform Cloud retains 14 days of audit log information. The audit trails API exposes a stream of audit events, which describe changes to the application entities (workspaces, runs, etc.) that belong to a Terraform Cloud organization. Audit trails are a paid feature that is available as part of the **Terraform Cloud for Business** upgrade package. Refer to the [Terraform Cloud pricing page](https://www.hashicorp.com/products/terraform/pricing) for more details. Unlike other APIs, the Audit Trails API does not use the [JSON API specification](/cloud-docs/api-docs/#json-api-formatting).
+Terraform Cloud retains 14 days of audit log information. The audit trails API exposes a stream of audit events, which describe changes to the application entities (workspaces, runs, etc.) that belong to a Terraform Cloud organization. Audit trails are a paid feature that is available as part of the **Terraform Cloud Business Tier** upgrade package. Refer to the [Terraform Cloud pricing page](https://www.hashicorp.com/products/terraform/pricing) for more details. Unlike other APIs, the Audit Trails API does not use the [JSON API specification](/cloud-docs/api-docs/#json-api-formatting).
```

Comment:
```suggestion
Terraform Cloud retains 14 days of audit log information. The audit trails API exposes a stream of audit events, which describe changes to the application entities (workspaces, runs, and other similar information) that belong to a Terraform Cloud organization. Audit trails are a paid feature that is available as part of the **Terraform Cloud Business Tier** upgrade package. Refer to the [Terraform Cloud pricing page](https://www.hashicorp.com/products/terraform/pricing) for more details. Unlike other APIs, the Audit Trails API does not use the [JSON API specification](/cloud-docs/api-docs/#json-api-formatting).
```

Suggest changing "etc." to "and other similar information" to avoid using Latin words and abbreviations.